cfpb/cfgov-refresh
# Add HMDA Explore page temporarily, with feature flag

Collaboration with @contolini

Add a static page that duplicates what is currently served at [cf.gov/hmda/explore](https://www.consumerfinance.gov/data-research/hmda/explore). In the near future, we will be removing the static site that is currently served by Apache at [cf.gov/hmda](https://www.consumerfinance.gov/data-research/hmda/).  This is the only page from that site we will still need.

With the `HMDA_LEGACY_REVIEW` flag, it will be served at `/data-research/hmda-new/explore`, so that stakeholders can review it on content.cf.gov. When we're ready to publish the page, (and when we've updated the Apache settings not to redirect traffic for that url), we'll switch to the `HMDA_LEGACY_PUBLISH` flag, and it will serve the page at `/data-research/hmda/explore`.


## Additions

- `hmda` app (we'll be adding more to this app later)
- URLs, view, and template for the HMDA Explore page

## Testing

1. Run this branch
2. In the Wagtail admin, update Wagtail Flags to enable the `HMDA_LEGACY_REVIEW` flag
3. Visit the new page at localhost:8000/data-research/hmda-new/explore
4. In the Wagtail admin, update Wagtail Flags to disable the `HMDA_LEGACY_REVIEW` flag and enable the `HMDA_LEGACY_PUBLISH` flag
5. Visit the page at localhost:8000/data-research/hmda/explore
    - The page will be the same at both URLs

The page should be identical to the page at https://www.consumerfinance.gov/data-research/hmda/explore EXCEPT for a few details:
- The new page removes the nav bar at the top
- The new page updates the notification box above the explorer tool to include a notification that the HMDA API will be deprecated

We have mockups if you want to see them.


## Notes

- This is a temporary addition. This will only exist until the Public Data Platform is disabled (later this year)

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
